snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/event/AbstractEvent.kt

```diff
@@ -22,10 +22,27 @@ import java.util.*
  * - "True" timestamp: user-defined custom event timestamp
  */
 abstract class AbstractEvent : Event {
-    /** List of custom contexts associated with the event.  */
-    @Deprecated("Old nomenclature.", ReplaceWith("entities"))
-    @JvmField
-    val customContexts: MutableList<SelfDescribingJson> = LinkedList()
+    private var _entities: MutableList<SelfDescribingJson> = LinkedList()
```

Comment:
Nit: this looks slightly more Kotlin like:

```suggestion
    private var _entities = mutableListOf<SelfDescribingJson>()
```